code/controller/TranslatableCMSTreeNodeExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @package translatable
+ */
+class TranslatableCMSTreeNodeExtension extends Extension {
+
+	function init() {
+		die("Init TranslatableCMSTreeNodeExtension. "); // this never gets called in the current config
```

Comment:
This is just a debug line in an unfinished PR: I did in fact want that line to be reached, but it never did and I couldn't yet figure out why. 

Due to this only being a "nice to have", I haven't yet been able to spend more time on it, but I still intend to later!..